Adyen/Core/AlreadyPaidPaymentComponent.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Provides a placeholder for payment methods that are already paid, just for display.
+/// :nodoc:
+public final class AlreadyPaidPaymentComponent: PaymentComponent {
```

Comment:
Is this used? I only see it checked on DropIn, but never created...

Reply:
its used in `OrderPaymentMethod`